project_management/turing_mybinder_role.md

```diff
@@ -0,0 +1,58 @@
+# Turing mybinder.org Cluster Admin Role
+
+This document outlines the minimal responsibilities and requirements for the technical administrator of the BinderHub deployment _The Turing Way_ donates to the [Binder Federation](https://blog.jupyter.org/the-international-binder-federation-4f6235c1537e).
+
+---
+
+## Context
+
+_The Turing Way_ supports and maintains a Kubernetes cluster that is part of the Binder Federation, facilitating the decentralisation of the mybinder.org service.
+This cluster also hosts Hub23, a private BinderHub for the Turing research community.
+Maintaining this infrastructure requires technical skill and a small time commitment that are detailed below.
+
+It is important to the Binder Operating team that we are not anonymous GitHub handles to one another.
+Knowing who we are and why we're here helps us support each other and connect those who can help each other.
+Please do say hi and [introduce yourself](https://discourse.jupyter.org/t/introduce-yourself/17)! :smile:
+
+## Requirements
+
+Ideally, the person in this role will be a member of the Turing Institute with an `@turing.ac.uk` domain email.
+This is due to the dependency on Turing internal processes/platforms, such as points 1 and 2 below.
+
+1) An account in the Turing's Azure tenancy
+2) Access to the `turingmybinder` subscription (can be arranged with IT Services through Top Desk)
+3) Knowledge of Kubernetes and Helm
+
+## Responsibilities
+
+Deployments are managed by CI in the [mybinder.org-deploy repo](https://github.com/jupyterhub/mybinder.org-deploy) and so day-to-day tasks are minimal and relatively non-disruptive.
+
+- Watch the [team-compass](https://github.com/jupyterhub/team-compass) for any relevant updates or opportunities for participation
+- Attend the [monthly team meetings](https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings.html) when able.
+  This does **not** have to be every month since the provided link also contains a monthly report archive for missed meetings.
```

Comment:
I was trying to make it explicit that attendance every month is not expected, since it isn't even expected of the JupyterHub/Binder team members (probably minced my words a bit on this). But turning up when you can and being engaged so we know who we all are is part of what makes it a fun team to be connected to.